firefox-ios/Client/Frontend/Browser/Toolbars/Redux/ToolbarMiddleware.swift

```diff
@@ -237,10 +237,6 @@ final class ToolbarMiddleware: FeatureFlaggable {
             scrollY: scrollOffset.y
         )
 
-        let needsAddressToolbarUpdate = toolbarState.addressToolbar.borderPosition != addressBorderPosition
-
-        guard needsAddressToolbarUpdate else { return }
-
```

Comment:
No longer need this check because now we only dispatch the action when the toolbar borders need updating